Tests/Shared/CustomAssertions/AssertAsync.swift

```diff
@@ -336,7 +336,7 @@ struct AssertAsync {
     }
 }
 
-@_functionBuilder
+@resultBuilder
```

Comment:
Isn't this Swift 5.4 only?

Reply:
Yep, hence it was failing in CI, so I removed it for now